### PR TITLE
adds missing rsync package to install on fresh f26

### PR DESCRIPTION
**Pull request by sahid**
Signed-off-by: Sahid Orentino Ferdjaoui <sahid.ferdjaoui@redhat.com>

### CODE REVIEW

File: docs/devenv-local.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Starting from a default Fedora 26 Server install the following
 extra software needs installing and configuring
 
 ```bash
-sudo dnf install etcd docker git libvirt-devel
+sudo dnf install etcd docker git libvirt-devel rsync
```
Comment (berrange) [inline docs/devenv-local.md:23]: I don't recall anything needing rsync ?  Could you say what/where you get an error without it ?

Comment (sahid) [reply docs/devenv-local.md:23]: When I started the script to build k8s - It seems to be a prerequisite [0]

[0] https://github.com/kubernetes/kubernetes/blob/master/build/common.sh#L167

Comment (berrange) [reply docs/devenv-local.md:23]: Ok I checked my test dev env and it seems that I got rsync installed by the Fedora installer when creating the guest. So I guess you must have selected different install options. Lets just add it here to be safe against this difference.

Comment (sahid) [reply docs/devenv-local.md:23]: Yes probably, for information I provisioned my environment with a Fedora 26 Server edition.

**Pull request merged**